x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/LifecyclePolicyTests.java

```diff
@@ -224,7 +224,7 @@ public static LifecyclePolicy randomTimeseriesLifecyclePolicy(@Nullable String l
                     frozenTime,
                     Collections.singletonMap(
                         SearchableSnapshotAction.NAME,
-                        new SearchableSnapshotAction(randomAlphaOfLength(10), randomBoolean())
+                        new SearchableSnapshotAction(randomAlphaOfLength(10), randomBoolean(), randomIntBetween(-1, 100))
```

Comment:
0 is not a valid setting (it would be like saying "you can allocate zero copies of the data to the node"), so I think this would be better as something like: 
```suggestion
                        new SearchableSnapshotAction(randomAlphaOfLength(10), randomBoolean(), (randomBoolean() ? null : randomIntBetween(1, 100)))
```